src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,68 @@
+package step2;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public int add(String text) {
+        if (text == null) {
+            return 0;
+        }
+
+        if (text.isEmpty()) {
+            return 0;
+        }
```

Comment:
```suggestion
        if (text == null || text.isBlank()) {
            return 0;
        }
```
예외 상황을 하나의 if문으로 합치는 것은 어떨까?